### ISSUE TITLE
Feature Request: Find Match Indicators in Preview Pane Vertical Scroll Bar

### ISSUE THREAD

**Issue by PaltryProgrammer** [dnGrep/dnGrep#658]
Kind Regards. Thank You for providing indicators in the Preview Pane Vertical Scroll Bar indicating the locations of the Search for text. May I please request also indicators in the Preview Pane Vertical Scroll Bar of text which is found via the Find command which I learned is activated via <Ctrl+F> Thank You Kindly

**Comment by doug24** [comment]
The ctrl-F find feature in the preview pane is part of the text control (AvalonEdit) used in dnGrep.  I will take a look into this, but I don't know if I can get file locations from that find operation back from the control in order to put them in the scroll bar.  And I don't want to try to reimplement that find just to add the indicators.

**Comment by doug24** [comment]
I did look into implementing this. The local file search in the preview pane is part of the AvalonEdit library, and all of the implementation is private.  So to do this would require changing the library code or reimplementing that search outside of AvalonEdit.  Neither option is good, so I'm going to say no to this one at this time.

**Comment by doug24** [comment]
I changed my mind on this one and have included it in release v3.0.64.  

Long story: This change required changes to a fork of AvalonEdit code. I had previously made some changes to support Right to Left Reading and raised a PR to get those changes in their release.  But I don't think they are ever going to support RTL in the product, so we'll have to keep the fork.  May as well make other improvements.

I added limits to the number of markers shown in the scrollbar to prevent problems in large files with a large number of matches.
